docs/api.md

```diff
@@ -390,7 +390,7 @@ The function is invoked after the document was created but before any of its scr
 #### page.exposeFunction(name, puppeteerFunction)
 - `name` <[string]> Name of the function on the window object
 - `puppeteerFunction` <[function]> Callback function which will be called in Puppeteer's context.
-- returns: <[Promise]> Promise which resolves with the result of `puppeteerFunction`.
```

Comment:
it does: https://github.com/GoogleChrome/puppeteer/blob/master/test/test.js#L761